tsl/test/sql/continuous_aggs_drop_chunks.sql

```diff
@@ -78,37 +79,49 @@ CREATE MATERIALIZED VIEW conditions_7
             SUM(value), COUNT(value)
         FROM conditions GROUP BY bucket WITH DATA;
 
-CREATE VIEW see_cagg AS SELECT * FROM conditions_7 WHERE bucket < 70 ORDER BY bucket;
+CREATE VIEW see_cagg AS SELECT * FROM conditions_7 WHERE bucket <= 70 ORDER BY bucket;
 
 SELECT * FROM see_cagg;
 
 -- This is the simplest case, when the updated bucket is inside a chunk, so it is expected
--- that the update is always reflected after the refresh on drop.
+-- that the update is refreshed.
 UPDATE conditions SET value = 4.00 WHERE time_int = 2;
 
 -- This case updates values in the bucket, which affects two different partials in
--- two different chunks, which are dropped in the same call to drop_chunks.
+-- two different chunks.
 UPDATE conditions SET value = 4.00 WHERE time_int = 9;
 UPDATE conditions SET value = 4.00 WHERE time_int = 11;
 
+SELECT _timescaledb_internal.refresh_continuous_aggregate('conditions_7', show_chunks('conditions', 20));
```

Comment:
```suggestion
CALL _timescaledb_internal.refresh_continuous_aggregate('conditions_7', show_chunks('conditions', 20));
```

Reply:
@mkindahl Will it work with `show_chunks`, which returns set of chunks? How will the procedure work with selecting all continuous aggregates of the hypertable?
Combining the refresh function with set of chunks and set of caggs is the reason to use the function and not the procedure. May be I miss something and it can be also simple with procedure?

Reply:
AFAICT, there is no real difference between functions and procedures internally, so I would be surprised if it did not work, but the only way to know for sure is to test it.

Reply:
The difference is in the usage. Procedures cannot be used in select statements, thus the query planner will not resolve calling show_chunks, since it returns a set. Your suggestion will require to use cursor, I guess.

Reply:
I tested it, and you're right. The set-returning function cannot be used with the procedure, but a function works.

Reply:
```
mats=# CREATE TABLE department(id int primary key, name text);
CREATE TABLE
mats=# CREATE TABLE employee(id int primary key, name text, salary int, departmentid int references department);
CREATE TABLE
mats=# INSERT INTO department VALUES (1, 'Management');
INSERT 0 1
mats=# INSERT INTO department VALUES (2, 'IT');
INSERT 0 1
mats=# INSERT INTO employee VALUES (1, 'John Smith', 30000, 1);
INSERT 0 1
mats=# INSERT INTO employee VALUES (2, 'Jane Doe', 50000, 1);
INSERT 0 1
mats=# INSERT INTO employee VALUES (3, 'Jack Jackson', 60000, 2);
INSERT 0 1
mats=# CREATE TABLE log(id int, message text);
CREATE TABLE
mats=# CREATE PROCEDURE LogName(int) AS $$
mats$#    INSERT INTO log VALUES ($1, format('%s was here', EmployeeName($1)))
mats$# $$ LANGUAGE sql;
CREATE PROCEDURE
mats=# CREATE FUNCTION EmployeeName(int) RETURNS text AS 'SELECT name FROM employee WHERE id = $1' LANGUAGE sql;
CREATE FUNCTION
mats=# CREATE FUNCTION GetEmployees() RETURNS SETOF int AS 'select id from employee;' language sql;
CREATE FUNCTION
mats=# CALL LogName(GetEmployees());
ERROR:  set-returning functions are not allowed in CALL arguments
LINE 1: CALL LogName(GetEmployees());
                     ^
mats=# DROP PROCEDURE LogName;
DROP PROCEDURE
mats=# CREATE FUNCTION LogName(int) RETURNS VOID AS $$
mats$#    INSERT INTO log VALUES ($1, format('%s was here', EmployeeName($1)))
mats$# $$ LANGUAGE sql;
CREATE FUNCTION
mats=# SELECT LogName(GetEmployees());
 logname 
---------
 [NULL]
 [NULL]
 [NULL]
(3 rows)

mats=# SELECT * FROM log;
 id |        message        
----+-----------------------
  1 | John Smith was here
  2 | Jane Doe was here
  3 | Jack Jackson was here
(3 rows)
```